elements/multicolumn/README.md

```diff
@@ -0,0 +1,54 @@
+Multicolumns are the main elements to create complex layouts. They can have multiple columns (we support max. 5 in our editor) and every columns can have children. The children array can contain every element except the full width container. [Read more](//fullwidth_container/README.md)
```

Comment:
We support max 5 in the edm editor. The format is prepared for infinite columns actually, and I believe you can have more columns in the chamaileon editor.

Reply:
Also, the children array of the columns! That is an important difference.